src/Components/Container.php

```diff
@@ -95,11 +97,30 @@ public function getOperation($need = TRUE)
     /**
      * Returns export component.
      * @param bool $need
-     * @return Export
+     * @return CsvExport
+     *
+     * @deprecated
      */
     public function getExport($need = TRUE)
     {
-        return $this->getComponent(Export::ID, $need);
+        $export = $this->getComponent(BaseExport::ID, $need);
+        if ($export) {
+            $export = $export->getComponent(CsvExport::CSV_ID, $need);
```

Comment:
Why only CSV export? Anything what is implementing Export should be able to be added as export